library/src/main/java/com/adevinta/android/barista/interaction/BaristaListInteractions.kt

```diff
@@ -53,9 +54,15 @@ object BaristaListInteractions {
   @JvmStatic
   @JvmOverloads
   fun clickListItemChild(@IdRes id: Int? = null, position: Int, @IdRes childId: Int) {
+    doOnListItemChild(id, position, childId, click())
+  }
+
+  @JvmStatic
+  @JvmOverloads
+  fun doOnListItemChild(@IdRes id: Int? = null, position: Int, @IdRes childId: Int, viewAction: ViewAction) {
     performMagicAction(id, position,
-        recyclerAction = actionOnItemAtPosition<ViewHolder>(position, clickChildWithId(childId)),
-        listViewAction = clickChildWithId(childId)
+        recyclerAction = actionOnItemAtPosition<ViewHolder>(position, onChildWithId(childId, viewAction)),
+        listViewAction = onChildWithId(childId, viewAction)
```

Comment:
Thank you for adding this functionality. As a user, what if I wanted to identify the child view using a ViewMatcher? Perhaps this function should use `onChildWithMatcher` instead of `onChildWithId` for greater flexibility from an end-user perspective?